frontend/next.config.ts

```diff
@@ -32,6 +32,20 @@ const nextConfig: NextConfig = {
   // https://nextjs.org/docs/app/api-reference/config/next-config-js/productionBrowserSourceMaps
   productionBrowserSourceMaps: true,
   serverExternalPackages: ['import-in-the-middle'],
+  async redirects() {
```

Comment:
Let's keep all redirects within [proxy](https://github.com/OWASP/Nest/blob/main/proxy/redirects.conf) for now -- please add them on nginx level.